### ISSUE TITLE
make the receiving page public by default

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#3705]
With #3873 we now have a Receiving page for Teams. However, we made it private to owners (and admins) because it's so rough. This ticket is about phasing it over to public by default. Steps:
- [ ] finish #3992—make the page something to be proud of
- [ ] add an option to make it public; private will be the default option for existing Teams
- [ ] make public the default for new Teams

---

**Original:** There's no way for anyone to see how much Teams are moving.
##

**Comment by mattbk** [comment]
Same as https://github.com/gratipay/gratipay.com/issues/3612?

**Comment by whit537** [comment]
Maybe? They were/are separate for ~users.

**Comment by mattbk** [comment]
~users now have Giving (giving) and History (receiving).
Teams could have Distributing (among team members), History (receiving), and Giving (to other teams).

**Comment by whit537** [comment]
> Giving (to other teams)

We don't have this right now, and I'm not sure we should. Let's discuss in a separate ticket, though.

**Comment by whit537** [comment]
History reflects all past activity in the `payments` and `exchanges` tables (and `transfers`, the old one). Giving and Receiving and Distributing are snapshots from the `payment_instructions` and `payroll` tables.

History is past, Giving, etc. is present/future.

**Comment by mattbk** [comment]
+1 from https://twitter.com/helloanselm/status/636484241560309760

**Comment by whit537** [comment]
> There's no way for anyone to see how much Teams are moving.

This immediate concern was addressed in #3721, though we should keep this ticket open since we still do want separate Receiving and "[Sharing](https://github.com/gratipay/inside.gratipay.com/issues/325)" pages that show more detail.

**Comment by whit537** [comment]
From @anselmh in #3757:

> Prior to the big change it was possible to see how many people participated per payout and also to see how many people gave how much money individually. I miss this option now. Is it gone forever, just coming back or what’s the status here?

**Comment by rohitpaulk** [comment]
I've started work on the receiving pages for teams at https://github.com/gratipay/gratipay.com/pull/3873. To convey the charging in arrears concept effectively, we'll need a proper overhaul of the way we present information - 

I'm thinking of something like this - (taken from [Harvest](http://getharvest.com/))

<img width="574" alt="gratipaydemo" src="https://cloud.githubusercontent.com/assets/3893573/11762963/7917adf2-a11e-11e5-9ea2-f496545cc938.png">

We'll have to tackle https://github.com/gratipay/gratipay.com/issues/3851 first though.

**Comment by rohitpaulk** [comment]
The way I see it, #3873 is good enough to merge - it solves problems that team owners and admins face. (https://github.com/gratipay/gratipay.com/issues/3779 and https://github.com/gratipay/gratipay.com/issues/3757 in particular)

Once that is done, we should move on to #3851, and then https://github.com/gratipay/gratipay.com/issues/3705#issuecomment-164167705

**Comment by whit537** [comment]
> Dues will be represented using a different color

+1; see #2832 for a previous (unlanded) example

**Comment by whit537** [comment]
We could also add "unfunded" in a third shade and it would help with #2534 ~= #2603.

**Comment by whit537** [comment]
I'm dropping "Distributing" tab from this ticket because that depends on #3433 before it makes sense.

**Comment by whit537** [comment]
Furthermore, with #3873 I'm refocusing this ticket to be about making the receiving tab public by default (cf. https://github.com/gratipay/gratipay.com/pull/3873#issuecomment-165444195 ff.). Right now it is admin/owner only. I'll update the ticket description with steps to cut over.

**Comment by whit537** [comment]
Dropping from Finish Team Basics. What we have with #3873 qualifies for "Basics."
